bin/zsnap

```diff
@@ -39,15 +40,27 @@ else:
 set_verbose(opts.verbose)
 
 snapshot_prefix = opts.prefix
-snapshot_postfix = lambda: time.strftime(opts.timeformat)
+snapshot_postfix = lambda: time.strftime(opts.timeformat,time.gmtime())
```

Comment:
Fair call, this was a hack I added & forgot to remove before making the PR. I think an option would be best as there are pros and cons to both (and the default must be preserved as you say)